scene/realistic-lighting-and-shadows/src/main/java/com/esri/samples/realistic_lighting_and_shadows/ComboBoxStringConverter.java

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright 2020 Esri.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.esri.samples.realistic_lighting_and_shadows;
+
+import javafx.util.StringConverter;
+
+import com.esri.arcgisruntime.mapping.view.LightingMode;
+/**
+ * Converts the LightningMode values to strings to display in the open ComboBox.
+ */
+public class ComboBoxStringConverter extends StringConverter<LightingMode> {
+
+  @Override
+  public String toString(LightingMode mode) {
```

Comment:
similar to my comments in LightingModeListCell.java: we should ensure we put in a null check here, and update the strings to the enum plain-english names.

```java

  @Override
  public String toString(LightingMode lightingMode) {
    if (lightingMode != null) {
      if (lightingMode == LightingMode.LIGHT) return "Light";
      else if (lightingMode == LightingMode.LIGHT_AND_SHADOWS) return "Light and shadows";
      else if (lightingMode == LightingMode.NO_LIGHT) return "No light";
    }
    return "";
  }
```